speechbrain/utils/fetching.py

```diff
@@ -58,38 +158,68 @@ def fetch(
     use_auth_token=False,
     revision=None,
     huggingface_cache_dir=None,
+    local_strategy: LocalStrategy = LocalStrategy.SYMLINK,
 ):
-    """Ensures you have a local copy of the file, returns its path
+    """Ensures you have a local copy of the file and returns its path.
+
+    If the file already exists under `<savedir>/<save_filename>` **and**
+    `overwrite == False`, then that path is always returned and no other action
+    is performed.
+
+    Unless the `local_strategy` is `LocalStrategy.NO_LINK`, the file will be
+    available under `<savedir>/<save_filename>`.
+
+    .. note::
+        If `source` is remote but available locally, `fetch` will not attempt
+        downloading again. When the `source` is the HF hub, it is OK to pass
+        `overwrite == True` to ensure you're getting the latest version, as the
+        HF Hub performs caching of its own.
+
+    Effect of local file strategies when the fetch source is a **local file**:
+
+    - `LocalStrategy.SYMLINK`: Create symlink to the file in the destination.
+    - `LocalStrategy.COPY`: Create copy of the file in the destination.
+    - `LocalStrategy.COPY_SKIP_CACHE`: Create copy of the file in the
+      destination.
+    - `LocalStrategy.NO_LINK`: Returns the path to the source file.
```

Comment:
There's many examples of backtick-ed code in this PR, I'm pretty sure sphinx documentation generation requires two backticks not just one.

But after more research it may be possible to change this behavior so that single backticks also renders as code:

https://github.com/sympy/sympy/issues/13519

Reply:
> There's many examples of backtick-ed code in this PR, I'm pretty sure sphinx documentation generation requires two backticks not just one.
> 
> But after more research it may be possible to change this behavior so that single backticks also renders as code:
> 
> [sympy/sympy#13519](https://github.com/sympy/sympy/issues/13519)

You're right, but I have actually set this a while back: https://github.com/speechbrain/speechbrain/blob/develop/docs/conf.py#L91

Lots of docstrings in SB inadvertently used single backticks which were just shown as italic, but always intended code formatting.

Reply:
Sweet, I wasn't aware this had happened but glad to see it!